src/diffusers/schedulers/scheduling_ddpm.py

```diff
@@ -119,6 +119,7 @@ def __init__(
         trained_betas: Optional[Union[np.ndarray, List[float]]] = None,
```

Comment:
Hmm, ok the changes are too big here. Let's just add a new scheduler class "unCLIPScheduler" for now. We can easily improve/change this is in the future. But the changes are too exotic here, so let's maybe just do a new scheduler here.

Reply:
sounds good to me!

Reply:
done here! https://github.com/huggingface/diffusers/pull/1428/commits/710f6482995a610b64c3132a609f99358bbe973e